benchmark/benchmarks_entrypoint.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import importlib.util
+import logging
+import os
+import sys
+
+
+class ImportModuleException(Exception):
```

Comment:
`ImportError` is native should work as well no?

Reply:
I don't get an `ImportError` because the function I use to import the module returns `None` when it can't import a given module afaiu.

Reply:
I can always throw an `ImportError` though